bdn_blocks.go

```diff
@@ -0,0 +1,37 @@
+package bloxroute_sdk_go
+
+import (
+	"context"
+
+	"github.com/bloXroute-Labs/gateway/v2/types"
+)
+
+// BdnBlockParams is the params object for the OnNewBlock subscription
+type BdnBlockParams struct {
+	// Include is the list of fields to include in the response.
+	// The values of these fields depend on the feed type.
+	// Optional (defaults to ["hash"])
+	Include []string `json:"include"`
+}
+
+// OnBdnBlock subscribes to a stream of all new blocks as they are propagated in the BDN.
+func (c *Client) OnBdnBlock(ctx context.Context, params *BdnBlockParams, callbackFunc CallbackFunc[*OnBdnBlockNotification]) error {
+	if params == nil {
+		params = &BdnBlockParams{}
+	}
+
+	if len(params.Include) == 0 {
+		params.Include = []string{"hash"}
```

Comment:
In both cloud-api/gateway, empty include by default will return all available fields ([link](https://docs.bloxroute.com/streams/bdnblocks)). Is this something we decided to change ? otherwise you might just keep it as empty slice

Reply:
Same for other feeds, newTx, TxReceipt etc.

Reply:
We decided to just include hash by default for now, but we might change it in the future if that what customers would want